src/CraftingPlan.js

```diff
@@ -1,21 +1,28 @@
 import React from 'react'
-import { Accordion, Segment, Header } from 'semantic-ui-react'
+import { Accordion, Button, Header, Segment } from 'semantic-ui-react'
+
 import CraftingCard from './CraftingCard'
 
-export default function CraftingPlan({items, dispatch}) {
+export default function CraftingPlan ({ items, dispatch }) {
 
   return (
     <Segment basic>
-      <Header>Crafting:</Header>
+      <Header
+        style={{
+          'max-width': '600px'
+        }}>
+        Crafting:
+        <Button content='Clear All' floated='right' size='mini' compact onClick={e => dispatch({ type: 'REMOVE_ALL' })} />
```

Comment:
what do you think of the basic button? (add basic as a property, like compact)

Reply:
I prefer the normal button with the grey background. The styled accordion already has the same look as the `basic` button and both of them together is too much negative white space for my taste.

But honestly, this is your project so you make the call :)
I'm just happy to contribute however I can. No need to consult me on design decisions.